src/utils/dataFetch.js

```diff
@@ -1,6 +1,6 @@
-import fetch from "isomorphic-fetch";
+import fetch from 'isomorphic-fetch';
```

Comment:
I think we should scrap this changes here since the next commit of lint fixes addresses to rever this change.

I am not sure why you would need to change double to single in a patchset and then single to double in another commit. Please remove this.

We would not need a commit which does break then another commit for fix than another as a hotfix for the previous two.

I would advise to squash the relevant commit so it will be better.